Zappy_IA_Report/NN/Code/Tutorial3.py

```
@@ -0,0 +1,37 @@
+
+test_loss, test_acc = model.evaluate(test_images, test_labels)
+
+print("Tested Acc:", test_acc)
+
+predictions = model.predict(test_images)
+
+print(predictions[0])
+print(class_names[np.argmax(predictions[0])])
+
+for i in range(5):
+    plt.grid(False)
```

Ruff (0.11.9)

Boolean positional value in function call

(FBT003)